### DISCUSSION TITLE
umi4如何开启webpack-bundle-analyzer ？

### DISCUSSION THREAD

**Discussion by kimmy-wang** [umijs/umi#9199] [Q&A]
umi4如何开启webpack-bundle-analyzer ？

**Comment by fz6m** [comment]
See https://umijs.org/docs/guides/env-variables#analyze

**Reply by kimmy-wang** [reply]
3q

**Reply by weilun0510** [reply]
为什么运行 `ANALYZE=1 umi dev` 之后，没看到包大小分析页面
![image](https://user-images.githubusercontent.com/50159515/199691325-7cc3eb9a-57ff-4fed-8d34-ffeff5e5110a.png)

**Reply by fz6m** [reply]
dev 时 webpack 不会进行产物优化，请以 `ANALYZE=1 umi build` 生产构建后的产物和分析为准。

**Reply by weilun0510** [reply]
有可视化页面吗

**Reply by Composur** [reply]
> See https://umijs.org/docs/guides/env-variables#analyze

请问不支持自定义配置了吗？
`analyze: {
    analyzerMode: 'server',
    analyzerPort: 8010,
    openAnalyzer: true,
    // generate stats file while ANALYZE_DUMP exist
    generateStatsFile: false,
    statsFilename: 'stats.json',
    logLevel: 'info',
    defaultSizes: 'parsed', // stat  // gzip
  }`
